tenants/ai-example/single-model-serving/base/main/create-ds-connection-secret.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: create-ds-connections
+  namespace: ai-example-single-model-serving
+spec:
+  selector: {}
+  template:
+    spec:
+      containers:
+        - args:
+            - -ec
+            - |-
+              echo -n 'Waiting for minio route'
```

Comment:
I would recommend setting up a new minio instance for each namespace instead of using the one from training.

I'm generally not a big fan of using minio across multiple namespaces unless you have a proper multi-tenant minio instance.  It just gets messy.

I went ahead and moved the minio instance I am using in dspa up to the components/apps folder to make it a bit more reusable:

https://github.com/redhat-ai-services/ai-accelerator/tree/main/components/apps/minio/base

And example of using it here:
https://github.com/redhat-ai-services/ai-accelerator/blob/9f1fde4da12e886af0836b431ade15dc101b21fc/tenants/ai-example/datasciece-pipelines/base/kustomization.yaml#L5

Reply:
fixed